src/application_config/application_mode_default.h

```diff
@@ -1,14 +1,18 @@
-#ifndef APPLICATION_MODE_DEFAULT_H
-#define APPLICATION_MODE_DEFAULT_H
```

Comment:
Keep include guard

Reply:
I didn't **remove**  
APPLICATION_MODE_DEFAULT_H
only  **moved** it up